pkg/rekor/extensions.go

```diff
@@ -0,0 +1,278 @@
+// Copyright 2022 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rekor
+
+import (
+	"encoding/asn1"
+	"slices"
+)
+
+var fulcioOIDPrefix = []int{1, 3, 6, 1, 4, 1, 57264}
+
+// createFulcioOID creates an extension with the Fulcio OID prefix (1.3.6.1.4.1.57264) and an extension input.
+func createFulcioOID(ext []int) asn1.ObjectIdentifier {
```

Comment:
Can you confirm if this function executes every time you read the global variable or if this executes just once? I think without an init() function, this would run each time, which would be slightly inefficient, though we should verify this. Not a big deal for this codebase, but I would lean towards making these statically declared even if there is repetition.

Reply:
this does require an init() function to execute once, thanks for pointing this out!